python/dask_cudf/dask_cudf/tests/test_applymap.py

```diff
@@ -0,0 +1,30 @@
+# Copyright (c) 2022, NVIDIA CORPORATION.
+
+import pytest
+from pandas import NA
+
+from dask import dataframe as dd
+
+from .utils import _make_random_frame
+
+
+@pytest.mark.parametrize(
+    "func",
+    [
+        lambda x: x + 1,
+        lambda x: x - 1,
+        lambda x: x + 0.5,
```

Comment:
Similar to previous suggestion -- if this retains similar coverage, feel free to simplify.
```suggestion
        lambda x: x - 0.5,
```